Kickstarter-iOS/Assets.xcassets/icons/heart-small-icon.imageset/Contents.json

```diff
@@ -0,0 +1,16 @@
+{
+  "images" : [
+    {
+      "idiom" : "universal",
+      "filename" : "icon-heart-filled.pdf",
```

Comment:
I need this icon for the `BackerDashboardProjectCell` and `BackerDashboardEmptyStateCell`.